app/views/npq_separation/admin/schools/index.html.erb

```diff
@@ -1,18 +1,49 @@
-<h1 class="govuk-heading-l">All schools</h1>
+<h1 class="govuk-heading-l">Workplaces</h1>
+<p class="govuk-body">Search for a workplace to find its:</p>
+<ul class="govuk-list govuk-list--bullet">
+  <li>ID</li>
+  <li>Unique Reference Number (URN)</li>
+  <li>UK Provider Reference Number (UKPRN)</li>
+  <li>local authority</li>
+  <li>address</li>
+</ul>
+
+<div style="background-color: #f3f2f1; padding: 20px 15px; margin-bottom: 30px;">
```

Comment:
Inline CSS will fail CSP checks, can this be moved to a class - I'm not clear on why this form needs different styling to the other filter forms though? Is it possible to re-use the class from those forms?